sdk/chaos/Azure.ResourceManager.Chaos/src/Generated/Models/OperationStatus.cs

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using Azure;
+
+namespace Azure.ResourceManager.Chaos.Models
+{
+    /// <summary> The status of operation. </summary>
+    public partial class OperationStatus
+    {
+        /// <summary> Initializes a new instance of OperationStatus. </summary>
+        internal OperationStatus()
+        {
+        }
+
+        /// <summary> Initializes a new instance of OperationStatus. </summary>
+        /// <param name="id"> The operation Id. </param>
+        /// <param name="name"> The operation name. </param>
+        /// <param name="startTime"> The start time of the operation. </param>
+        /// <param name="endTime"> The end time of the operation. </param>
+        /// <param name="status"> The status of the operation. </param>
+        /// <param name="error"> The error detail of the operation if any. </param>
+        internal OperationStatus(string id, string name, string startTime, string endTime, string status, ErrorResponse error)
+        {
+            Id = id;
+            Name = name;
+            StartTime = startTime;
+            EndTime = endTime;
+            Status = status;
+            Error = error;
+        }
+
+        /// <summary> The operation Id. </summary>
+        public string Id { get; }
+        /// <summary> The operation name. </summary>
+        public string Name { get; }
+        /// <summary> The start time of the operation. </summary>
+        public string StartTime { get; }
+        /// <summary> The end time of the operation. </summary>
+        public string EndTime { get; }
+        /// <summary> The status of the operation. </summary>
+        public string Status { get; }
+        /// <summary> The error detail of the operation if any. </summary>
+        internal ErrorResponse Error { get; }
```

Comment:
@ArthurMa1978 We are using the common-types errorResponse, as directed, but this is creating errors that block build. Is there something we should be doing differently, or is this potentially an AutoRest bug?

Reply:
Basically, we end up with two conflicting `Error` properties in this `OperationStatus` class.

Reply:
the relevant swagger spec is here https://github.com/Azure/azure-rest-api-specs/blob/5ae522bc106bf8609c6cb379e584aa3e0e2639f3/specification/chaos/resource-manager/Microsoft.Chaos/stable/2023-11-01/operationStatuses.json#L113C11-L113C11

Reply:
we addressed the issue. it turns out to be an issue with our openapi spec